### DISCUSSION TITLE
Add furigana/ruby characters support

### DISCUSSION THREAD

**Discussion by TheHanziChauvanist** [quarto-dev/quarto-cli#9367] [Feature Requests]
### Description

I think that we should add furigana/ruby support into Quarto, and it needs to be East Asian language agnostic, which means that we can create Arabic/English furigana as well as Japanese/English furigana. The syntax can be taken from here:

https://github.com/iltrof/furigana-markdown-it

**Comment by mcanouil** [comment]
There is at least one Pandoc filter that you can use for that: <https://github.com/waldeir/pandoc-filter-furigana>.

````yaml
filters:
  - handleFurigana.py
````

You can also adapt or create your own filter and packaged it as a Quarto extension: https://quarto.org/docs/extensions/filters.html
